package.json

```diff
@@ -5,10 +5,10 @@
   "main": " ",
   "dependencies": {
     "husky": "^3.0.5",
-    "npm-run-all": "^4.1.5"
-  },
-  "devDependencies": {
-    "standard-version": "^5.0.0"
+    "lodash": "^4.17.15",
+    "lodash.template": "^4.5.0",
+    "npm-run-all": "^4.1.5",
+    "standard-version": "^7.0.0"
```

Comment:
standard-version and npm-run-all belong to devDependencies

Reply:
Det er litt vanskelig å skille mellom dev og prod dependencies fordi den pakke er bare dev verktøy her. hvis det er bare dev prosjekt, kanskje vi kan holde alle dependencies som default